target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -144,6 +144,7 @@ ramips_setup_interfaces()
 	allnet,all0256n-8m|\
 	allnet,all5002|\
 	allnet,all5003|\
+	apexera,netbot|\
```

Comment:
Can you explain why you need this:
		ucidef_add_switch "switch0"
		ucidef_add_switch_attr "switch0" "enable" "false"
?